scenarios/anomaly-detection/gradle.properties

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+### dependencies
+flinkConnectorVersion=0.3.0-102.0e30f47-SNAPSHOT
```

Comment:
Do we need a new file at the module level? Can we not make use of the root gradle properties file?

Reply:
Removed `gradle.properties` from scenarios sub-projects. Now all properties are in the root gradle properties file.